docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-snowflake-loader/index.md

```diff
@@ -11,4 +11,6 @@ The Snowflake Loader consists of two independent applications:
 
 Both applications communicate through DynamoDB table, called "processing manifest" and used to maintain pipeline state. Both applications use same self-describing JSON configuration file of [schema](https://raw.githubusercontent.com/snowplow/iglu-central/master/schemas/com.snowplowanalytics.snowplow.storage/snowflake_config/jsonschema/1-0-3) `com.snowplowanalytics.snowplow.storage/snowflake_config/jsonschema/1-0-3`, which contain both transformer- and loader-specific properties.
 
+Type reference for our loaders could be found [here](/docs/understanding-tracking-design/json-schema-type-casting-rules).
```

Comment:
Does the table apply to both this loader and RDB loader? In any case, I would just drop this from here because I’ve moved the Snowflake loader to legacy apps (as you will discover upon rebasing).

Reply:
Same applies.